common/scripts/ci/run-sdk-backend-tiger-integration-tests-with-live-backend.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash -i
+set -ex
+
+export EXECUTOR_NUMBER=1
+
+DIR=$(echo $(cd $(dirname "${BASH_SOURCE[0]}") && pwd -P))
+ROOT_DIR="${DIR}/../../../"
+NODE_IMAGE='020413372491.dkr.ecr.us-east-1.amazonaws.com/tools/gdc-frontend-node-16:node-16.13.0-yarn-1.22.17'
+DATA_LOADER_IMAGE='registry.gitlab.com/gooddata/gdc-nas/data-loader:master'
+
+export IMAGE_ID=gooddata-cn-ce-aio-${EXECUTOR_NUMBER}
+export NETWORK_ID=aio-net-${EXECUTOR_NUMBER}
+export PORT_NUMBER=300${EXECUTOR_NUMBER}
+export HOST=http://localhost:$PORT_NUMBER
+export TEST_HOST=http://$IMAGE_ID:$PORT_NUMBER
```

Comment:
`HOST` and `TEST_HOST` must be flexible to be able to run against both AIO and Tiger/Panther